src/framework/preferences/js/Builder.js

```diff
@@ -17,37 +17,76 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
 (function ($, fluid) {
     "use strict";
 
-    fluid.registerNamespace("fluid.prefs");
+    fluid.registerNamespace("fluid.prefs.builder");
+
+    /**
+     * A merge policy where the items are not merged, but returned as a merge array to be handled by another
+     * function (e.g. by an expander).
+     *
```

Comment:
Please could you write a comment explaining where such mergePolicies come from (a similar one was called fluid.membersMergePolicy in the old framework, and then later fluid.deferringMergePolicy post FLUID-6145), and also that they are the result of a bug now written up as https://issues.fluidproject.org/browse/FLUID-6457